src/helpers/analytics.coffee

```diff
@@ -0,0 +1,64 @@
+_ = require 'underscore'
+Router = require 'react-router'
+{HistoryLocation} = require 'react-router'
+DestinationHelper = require '../helpers/routes-and-destinations'
+
+# generate custom event data for routes
+Events =
+  viewTaskStep: ({courseId}) ->
+    # track all work done on a course
+    Analytics.sendEvent 'Course', 'Work', label: courseId
+
+  viewStudentDashboard: ({courseId}) ->
+    # compare activity between courses
+    Analytics.sendEvent 'Student', 'Dashboard', label: courseId
+
+
+# Translators convert a url like '/foo/bar/123/baz/1' into a simplified one like just '/foo/bar'
+Translators =
```

Comment:
Ah, yeah I see your point.  Yes we could use that but we probably should clean them up.  For instance the `dashboard` name isn't actually the dashboard, it's the course chooser.  

I'm thinking that the end-user of the analytics might not find it very user-friendly to see "calendarViewPlanStats".  They'd have to figure out that that corresponds to the quick metrics modal.

Google Analytics has a "drill down" view that allows viewing all hits for url segments.  with the pseudo url scheme I've setup they could view all hits for 'teacher' , page type, and then course.  We shouldn't be able to easily support that if we just logged the name.